web/app/testUtils/mockStyles.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
What does this file do?

Reply:
Mocks imports of scss files during the tests

Reply:
it is noop for importing styles in jest environment